src/eth_plugin_interface.h

```diff
@@ -7,30 +9,43 @@
 #include "tokens.h"
 #include "shared_context.h"
 
-// Interface version. To be updated everytime we introduce breaking changes to the plugin interface.
-typedef enum {
+/*************************************************************************************************
+ * Comments provided in this file are quick reminders on the usage of the plugin interface       *
+ * Reading the real plugin documentation is GREATLY recommended.                                 *
```

Comment:
we have also a good doc available on dev portal --> section : `plugin `and then `code-walkthrough`
where all the steps are described

Reply:
This is more generic, it has more its place in the README than in the comments on the interface file ?